unit_tests/ut_h_stats.amlg

```diff
@@ -348,8 +348,7 @@
 		result
 			(call_entity "howso" "get_prediction_stats" (assoc
 				"robust" (false)
-				"stats" (list "accuracy" "precision")
-				"holdout" (true)
+				"stats" (list "accuracy" "precision" "confusion_matrix")
```

Comment:
did you intend to remove the 'holdout' parameter here? that was being tested here per the comment prior to the assert_approximate, if it's removed, don't we expect accuracy and precision scores to be different? if they are still the same, was the holdout not being tested correctly?

Reply:
The holdout parameter was a leftover paramter that I forgot to remove from the test. It was unused.

Reply:
done